digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ViiteVkmClient.scala

```diff
@@ -28,6 +30,33 @@ class ViiteVkmClient {
 
   private val client = HttpClientBuilder.create().build
 
+  def get(path: String, params: Map[String, String]): Either[Any, VKMError] = {
+
+    val url =
+      if(params.nonEmpty)
+        getRestEndPoint + path + "?"+params.map(p => if (p._2.nonEmpty) p._1 + "="+ p._2).mkString("&")
```

Comment:
Can you use URIBuilder to build url
See for example https://stackoverflow.com/questions/9907161/commons-httpclient-adding-query-string-parameters-to-get-post-request